app/assets/stylesheets/overrides.scss

```diff
@@ -0,0 +1,6 @@
+// Any SCSS in this file will be loaded last in the pipeline
+.media,
+.media-body {
+  overflow: visible;
+  zoom: 1;
+}
```

Comment:
Can we do this in a more specific way to reduce the likelihood of unexpected side effects